app/controllers/api/v1/packages_controller.rb

```diff
@@ -388,6 +409,10 @@ def existing_package
         end
       end
 
+      def assign_detail
```

Comment:
if you can get accepts_attributes_for working correctly, then you can remove this method.